scripts/rmc-rustc

```diff
@@ -41,6 +41,7 @@ set_rmc_lib_path() {
             echo "ERROR: Could not find RMC library."
             echo "Looked for: \"$REPO_DIR/target/*/librmc.rlib\""
             echo "Was RMC library successfully built first?"
+            echo "Try running <RMC_ROOT>/scripts/setup/build_rmc_lib.sh"
```

Comment:
Should the line above ("Was RMC library successfully built first?") be removed? It obviously wasn't, hence the newly added guidance.

Reply:
Good point, will remove that.